v2/compiler/argo.go

```diff
@@ -68,6 +76,12 @@ func Compile(job *pipelinespec.PipelineJob, opts *Options) (*wfapi.Workflow, err
 		if opts.LauncherImage != "" {
 			compiler.launcherImage = opts.LauncherImage
 		}
+		if opts.PipelineRoot != "" {
+			if job.RuntimeConfig == nil {
+				job.RuntimeConfig = &pipelinespec.PipelineJob_RuntimeConfig{}
+			}
+			job.RuntimeConfig.GcsOutputDirectory = opts.PipelineRoot
```

Comment:
Can PipelineRoot be something other than GCS path? Should we set GcsOutputDirectory if pipelineRoot is minio or other platform?